ocs_ci/ocs/dr/dr_workload.py

```diff
@@ -151,9 +151,18 @@ def __init__(self, **kwargs):
         self.drpc_yaml_file = os.path.join(
             self.workload_subscription_dir, self.workload_name, "drpc.yaml"
         )
+        self.drpc_yaml_file_placement = os.path.join(constants.DRPC_PATH)
         self.channel_yaml_file = os.path.join(
             self.workload_subscription_dir, "channel.yaml"
         )
+        workload_details = kwargs.get("workload_details")
+        self.is_placement = workload_details.get("is_placement")
+        if self.is_placement is not None:
```

Comment:
if self.is_placement should work

Reply:
yes, it impacts functionality. this block will execute if self.is_placement is anything except **None** which means even self.is_placement is False